htconfig.php

```diff
@@ -1,112 +0,0 @@
-<?php
-
-// If automatic system installation fails:
-
-
-die('The configuration you did manually contains some mistakes. Please have a look at your .htconfig.php file.');
```

Comment:
Is this fallback for mis-configuration by copy-and-pasting to prevent dead nodes without a hint of why it is dead still around somehow?

I agree that it should not be needed, but people are that lazy hence it is needed.

Reply:
If the sample config file is copied to `local.ini.php` without editing, then the database connection will fail, which will show the laconic "This site is unavailable at the moment". The `System::unavailable()` should be reworked, maybe in a separate PR?

Reply:
This "hack" was there especially for those type of people.

Maybe a similar `die()` statement at the top of the new config file template. It still is a PHP file that can die with a hint.

Reply:
I'd rather rework the default error page.

Reply:
Could you please do it with this PR then.